artifactory/1.11/artifactory-enterprise.md

```diff
@@ -0,0 +1,142 @@
+
+```
+sudo mount artifactoryha.mount.com:/artifactory /var/artifactory/
+```
+
+You must ensure this mount is writeable.
+
+### Setting up Artifactory Enterprise
+
+1. Create a new file called `artifactory-enterprise-options.json` with the
+   following content.
+
+Be sure to:
+
+- replace `service.licenses` with your own license string. Note: Licenses can be configured in the Artifactory UI.
+- replace `service.database.user` and `service.database.password` with the
+  correct credentials if you have customised these values.
+- replace `artifactory` within `service.database.url` with the correct database name
+  if you have used a different one.
+- replace `enterprise.master_key` with your own master key value.
+
+```
+{
+  "service": {
+    "name": "artifactory",
+    "licenses": "replaceme",
+    "host-volume": "/var/artifactory",
+    "database": {
+      "type": "postgresql",
+      "host": "postgresql.marathon.mesos",
+      "port": 5432,
+      "url": "jdbc:postgresql://postgresql.marathon.mesos:5432/artifactory",
+      "user": "jfrogdcos",
+      "password": "jfrogdcos"
+    }
+  },
+  "enterprise": {
+    "enabled": true,
+    "master_key": "replaceme"
+  }
+}
+```
+
+
+2. Run the following DC/OS CLI command to install Artifactory Enterprise:
+
+```
+dcos package install --options=artifactory-enterprise-options.json artifactory
+```
+
+3. Check that Artifactory is up and running successfully by checking the "Services" tab of DC/OS.
+
+## Install Artifactory-lb
+
+Once Artifactory is up and running, [follow this guide to set up
+Artifactory-lb](artifactory-lb.md).
+
+## Scaling Artifactory Enterprise
+
+To make Artifactory Enterprise highly available, you need to add licenses via
+the UI and install a "secondary":
+
+1. Add more licenses for secondary nodes in Artifactory UI:
+
+![Add More Licenses](img/add_licenses.png)
+
+
+2. Create a new file called `artifactory-enterprise-secondary-options.json` with
+   the following content.
+
+Be sure to:
+
+- replace `service.licenses` with your own license string (only one node's
+  license is required here, the rest can be configured in the Artifactory UI)
+- replace `service.database.user` and `service.database.password` with the
+  correct credentials if you have customised these
+- replace `artifactory` within `service.database.url` with the correct database name
+  if you have used a different one
+- replace `enterprise.master_key` with key specified for primary artifactory instance.
+
+```
+{
+  "service": {
+    "name": "artifactory",
+    "licenses": "replaceme",
+    "host-volume": "/var/artifactory-secondary",
+    "database": {
+      "type": "postgresql",
+      "host": "postgresql.marathon.mesos",
+      "port": 5432,
+      "url": "jdbc:postgresql://postgresql.marathon.mesos:5432/artifactory",
+      "user": "jfrogdcos",
+      "password": "jfrogdcos"
+    }
+  },
+  "enterprise": {
+    "enabled": true,
+    "master_key": "replaceme",
```

Comment:
Still not sure what it wants for a master_key, and I can't just delete it like I did in the earlier JSON file, this one actually causes a failure to deploy after:

Caused by: java.lang.RuntimeException: Timed out waiting for master.key file to be made available at /opt/jfrog/artifactory/etc/security/master.key